pickem/pickem_homepage/templatetags/pickem_homepage_extras.py

```
@@ -234,6 +275,21 @@ def is_game_locked(game):
+def is_game_locked_for_pool(game, pool):
+    """Check if picks are locked for a game using the current pool settings."""
+    try:
+        from pickem.utils import is_pick_locked_for_pool
+        week_games = GamesAndScores.objects.filter(
+            gameseason=game.gameseason,
+            gameWeek=game.gameWeek,
+            competition=game.competition
+        )
+        is_locked, _lock_reason = is_pick_locked_for_pool(game, pool, week_games)
+        return is_locked
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)